packages/liveblocks-react/src/index.test.tsx

```diff
@@ -17,6 +17,16 @@ import * as React from "react";
 
 import { createRoomContext } from "./factory";
 
+type TestID = "me-x" | "increment" | "othersJson" | "liveObject" | "unmount";
+
+function testId(testId: TestID) {
+  return testId;
+}
+
+function element(testId: TestID) {
+  return screen.getByTestId(testId);
```

Comment:
Light wrapper around `.getByTestId()` that enforces a type check on the literal.